pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -106,14 +108,32 @@ func desiredDNSConfigMap(dns *operatorv1.DNS, clusterDomain string) (*corev1.Con
 		clusterDomain = "cluster.local"
 	}
 
+	upstreamResolvers := operatorv1.UpstreamResolvers{
+		Upstreams: []string{"/etc/resolv.conf"},
+		Policy:    operatorv1.SequentialForwardingPolicy,
+	}
+
+	if dns.Spec.UpstreamResolvers.Upstreams != nil && len(dns.Spec.UpstreamResolvers.Upstreams) > 0 {
+		upstreamResolvers.Upstreams = dns.Spec.UpstreamResolvers.Upstreams
+		if len(dns.Spec.UpstreamResolvers.Upstreams) == 1 && dns.Spec.UpstreamResolvers.Upstreams[0] == "" {
```

Comment:
Why would this happen? Why would the user specify an array with the first item being an empty string.

Reply:
A scenario might be a misinterpretation from a user who wants to simply modify the  policy, so  instead of omitting the upstreams completely, would set it empty. 
Maybe far fetched? 
If you all think that, let's remove it